File: Projeto B/src/repositories/folha_repository.ts
import { Folha } from "../models/folhamodel";

const folhas: Folha[] = [];

export class FolhaRepository {
  cadastrar(folha: Folha) : Folha[] {
    folhas.push(folha);
    return folhas;
  }

  listar() : Folha[] {
    return folhas;
  }
}

File: Projeto A/Folha_envio/Repository/folha_repository.cs
using ProjetoA.Models;
using System.Collections.Generic;

namespace ProjetoA.Repository
{
    public class FolhaRepository
    {
        private readonly List<Folha> folhas = new List<Folha>();

        public List<Folha> Cadastrar(Folha folha)
        {
            folhas.Add(folha);
            return folhas;
        }

        public List<Folha> Listar()
        {
            return folhas;
        }
    }
}
